alert_source.go

```diff
@@ -0,0 +1,57 @@
+package opslevel
+
+import (
+	"github.com/shurcooL/graphql"
+)
+
+type AlertSourceExternalIdentifier struct {
+	Type 			 AlertSourceTypeEnum	`graphql:"type" json:"type"`
+	ExternalId string 							`graphql:"externalId" json:"externalId"`
+}
+
+type AlertSource struct {
+	Name           string
```

Comment:
because this struct is used for unmarshaling the response data - for safety incase we need to change the field name - its best to put the `graphql` struct tags on these ones with the exact caseing used in our API responses.  Looking back I didn't do this with the User example I gave you and that was my bad - but it really should be there for safety (even if it works correctly now because it downcases the first letter).